src/main/java/de/tum/in/www1/artemis/domain/enumeration/ProjectType.java

```diff
@@ -5,5 +5,5 @@
  * Differentiates different templates for a programming language.
  */
 public enum ProjectType {
-    MAVEN, ECLIPSE
+    MAVEN, ECLIPSE, PLAIN, XCODE
```

Comment:
I would prefer PLAIN and XCODEPROJECT, since otherwise for me it connotes that PLAIN is independent of Xcode

```suggestion
    MAVEN, ECLIPSE, PLAIN, XCODEPROJECT
```

Reply:
my idea was to let PLAIN be independent. At some point we might have a similar setup of a programming language, which might need a PLAIN setup. 
As an example for Javascript, the project type might be PLAIN, REACT, Angular.

Reply:
Alright, I see your point. For me XCODE is fine too.